docs/contributing/docs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Crossplane Documentation"
```

Comment:
We typically try to wrap docs at 80 characters to make it easier to comment on smaller sections. Would you be open to doing that in this doc (and potentially add it as part of this guide as well)?

Reply:
I don't think this works well for text and makes it harder to read in an editor. 

Not only is it unlikely to be viewed on a terminal where the line limit makes sense but it becomes very messy when you're including links. [For example](https://github.com/crossplane/crossplane/blob/432981c7e27063f89376421189d20b4acb7e643d/docs/guides/argo-cd-crossplane.md?plain=1#L18) the link alone is 70 characters.

Reply:
> I don't think this works well for text and makes it harder to read in an editor.

We have had pretty strong consensus on doing this in the past due to the GitHub suggestion granularity, but I am inclined to defer to the person who literally writes documentation for a living :) If there is strong push back we can always change in the future, and we can preserve the practice of wrapping in design docs for now where we are perhaps going to give more granular feedback.

Reply:
+1 to being willing to change in the future and thanks for being flexible.